actionview/lib/action_view/template.rb

```diff
@@ -324,8 +329,10 @@ def handle_render_error(view, e) #:nodoc:
       end
 
       def locals_code #:nodoc:
+        locals = @locals.delete_if { |key| RUBY_KEYWORDS.include?(key.downcase) }
```

Comment:
I'm just unclear why those would give the same error.

If the error they're giving is a more general "you can't set a constant here", then it seems we'd want to handle that separately -- `Foo` would be just as problematic as `Alias`.

Do we want to check for something like `/\A(?!=[A-Z0-9])\w+\z/`? (There's probably a better definition of a valid local identifier than that, somewhere.)